source/base/parameter_handler.cc

```diff
@@ -1409,7 +1408,41 @@ bool ParameterHandler::read_input_from_string (const char *s)
 {
   // create an istringstream representation and pass it off
   // to the other functions doing this work
-  std::istringstream in (s);
+
+  // concatenate the lines ending with "\"
+
+  std::ifstream file(s);
+  std::string str;
+  std::string file_contents;
```

Comment:
This changes the meaning of the function. It used to just get the *contents* of a file as a string, but now it gets a *file name*.